python/tvm/relay/backend/profiler_vm.py

```diff
@@ -49,8 +50,9 @@ def __init__(self):
         self.mod = _vm._VMCompilerProfiler()
         self._compile = self.mod["compile"]
         self._get_vm = self.mod["get_vm"]
+        self._set_params_func = self.mod["set_params"]
 
-    def compile(self, mod, target=None, target_host=None):
+    def compile(self, mod, target=None, target_host=None, params=None):
```

Comment:
Can we reuse refactor the VMCompiler to reduce the redundent code between VMCompilerProfiler and VMCompiler

Reply:
done